### PR TITLE
BUILD-1287 Add .github/CODEOWNERS

### PR DESCRIPTION
**Pull request by Godin**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+.github/CODEOWNERS @SonarSource/languages-team-dotnet
```
Comment (costin-zaharia-sonarsource) [inline .github/CODEOWNERS]: If I remember correctly this tool is used by the sonar lint team too. @duncanp-sonar would you like to be added as an owner too?

**Review by costin-zaharia-sonarsource — CHANGES_REQUESTED** [review]
Could you please add @duncanp-sonar and @rita-sonarsource too?

**Comment by Godin** [comment]
@costin-zaharia-sonarsource done

**Review by costin-zaharia-sonarsource — APPROVED** [review]
LGMT!

**Pull request merged**